certified-connectors/Powell Teams/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Powell.Teams.Web.PublicAPI",
```

Comment:
@PowellSoftware The connector's title looks like an operation id or application name, Can you please provide valid title to your connector, like below:
`"title": "Powell Teams"`

Also, can you please make sure that you are following all the instructions mentioned in the [documentation](https://docs.microsoft.com/connectors/custom-connectors/certification-submission) to meet all the standards for your connector.